src/components/Questions/QuestionModal.vue

```diff
@@ -301,11 +310,11 @@ To attempt Q.${props.currentQuestionIndex + 1}, unselect an answer to another qu
         // answer: CT
         // updated answer -> [AQ, BR, CS, CT]
         // if answer: CS, updated answer -> [AQ, BR]
-        if (state.draftResponses[props.currentQuestionIndex] == null) {
-          state.draftResponses[props.currentQuestionIndex] = []
+        if (state.draftResponses[shuffledQuestionIndex.value] == null) {
```

Comment:
this is a bit weird.. u could've just sent shuffledQuestionIndex as props.. just like currentQuestionIndex ..